spring-context-support/src/main/java/org/springframework/cache/ehcache/EhCacheCacheManager.java

```diff
@@ -49,10 +49,21 @@ public EhCacheCacheManager() {
 	}
 
 	/**
-	 * Create a new EhCacheCacheManager for the given backing EhCache CacheManager.
+	 * Create a new non-transaction-aware EhCacheCacheManager for the given backing EhCache CacheManager.
 	 * @param cacheManager the backing EhCache {@link net.sf.ehcache.CacheManager}
 	 */
 	public EhCacheCacheManager(net.sf.ehcache.CacheManager cacheManager) {
+		this(false, cacheManager);
```

Comment:
While I am not denying the setter is confusing, this is not the scope of this PR to change that.